CODE_OF_CONDUCT.md

```diff
@@ -50,7 +50,7 @@ decisions when appropriate.
 
 ## Scope
 
-This Code of Conduct applies within all community spaces, and also applies when
+This Code of Conduct applies within all community spaces and also applies when
```

Comment:
Removing the comma here is not wrong, but leaving it there isn't wrong either.
It is mainly here to add a pause in a longer sentence to improve readability.

This text is from the upstream covenant and thus recommended not to change it. Please revert this change.

Reply:
Unresolving, as this comment was marked resolved without an action that resolved the comment.